test/src/main/java/io/strimzi/test/k8s/KubeClusterResource.java

```diff
@@ -241,6 +260,18 @@ public void deleteCustomResources(String... resources) {
         }
     }
 
+    public void deleteCustomResources(ExtensionContext extensionContext, String... resources) {
+        final String namespaceName = !extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).get("NAMESPACE_NAME").toString().isEmpty() ?
+            extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).get("NAMESPACE_NAME").toString() :
+            getNamespace();
+
+        for (String resource : resources) {
```

Comment:
What is in resources? yaml contents or paths?

Reply:
`@param resources array of paths to yaml files with resources specifications`